settings.gradle.kts

```diff
@@ -16,21 +16,22 @@ pluginManagement {
         kotlin("plugin.serialization") version kotlinVersion
         kotlin("native.cocoapods") version kotlinVersion
         id("com.squareup.sqldelight") version sqldelightVersion
-        id("org.jlleitschuh.gradle.ktlint") version "10.2.1"
+        id("org.jlleitschuh.gradle.ktlint") version "11.0.0"
+        id("org.jetbrains.compose") version "1.2.0"
     }
 
     resolutionStrategy {
         eachPlugin {
             if (requested.id.namespace == "com.android" || requested.id.id == "android-gradle") {
-                useModule("com.android.tools.build:gradle:7.0.4")
+                useModule("com.android.tools.build:gradle:7.2.1")
```

Comment:
get a warning about this version not being tested on android 33. Looks like you can update to 7.3.0

Reply:
It seems in Android studio I cannot use a version above 7.2.1, it shows error `The project is using an incompatible version (AGP 7.3.0) of the Android Gradle plugin. Latest supported version is AGP 7.2.1`